force-app/main/default/classes/DMLOperations.cls

```diff
@@ -48,27 +53,37 @@ public with sharing class DMLOperations {
      */
     public static void createAccount(String name, String industry) {
         // Create a new Account and set the fields inline
-
+        Account acc2 = new Account(Name = name, Industry = industry);
+            
         // Insert the Account
+        insert acc2;
     }
 
     
     /**
      * Question 3
-     * Create a new Contact and link it to the Account id provided then inserts it into the database.
+     * Create a new Contact and link it to the Account id provided, then inserts it into the database.
      * @param accountId The Id of the Account to link the Contact to.
      * @return The Id of the newly created Contact.
      */
     public static Id insertNewContact(Id accountId) {
+
         // Create a new Contact instance
+        Contact con = new Contact();
 
         // Set some fields on the Contact
+        con.LastName = 'Rollins';
+        con.FirstName = 'Henry';
+        con.email = 'henry@blackflag.com';
+
         // Set the AccountId field to the given accountId
+        con.AccountId = accountId;
 
         // Insert the Contact into the database
+        insert con;
```

Comment:
Try to see if the user has the permission to insert an account https://salesforce.stackexchange.com/questions/92209/how-to-check-access-of-user-for-particular-user-on-object